stable/k8s-spot-termination-handler/templates/daemonset.yaml

```diff
@@ -65,3 +69,8 @@ spec:
     {{- if .Values.priorityClassName }} 
       priorityClassName: {{ .Values.priorityClassName }}
     {{- end }}
+  {{- if .Values.updateStrategy }}
+  updateStrategy:
+    type: {{ .Values.updateStrategy }}
```

Comment:
please add the options for the rolling update as well.
for example
```
    rollingUpdate:
      maxUnavailable: 1
```
```

Reply:
Done.